sdk/core/Azure.Core/src/Shared/ExponentialDelayStrategy.cs

```diff
@@ -13,7 +13,7 @@ namespace Azure.Core
     /// the sequence {1, 1, 1, 2, 4, ...32}.
     /// </summary>
     /// <remarks>Polling interval always follows the given sequence.</remarks>
-    internal class ExponentialDelayStrategy : DelayStrategy
+    internal class SequentialDelayStrategy : DelayStrategy
```

Comment:
Is this intentional? We already have a `SequentialDelayStrategy` below.

Reply:
Will delete.